ext/fileinfo/libmagic/apprentice.c

```diff
@@ -2909,7 +2909,7 @@ apprentice_map(struct magic_set *ms, const char *fn)
 	if (dbname == NULL)
 		goto error;
 
-		stream = php_stream_open_wrapper((char *)fn, "rb", REPORT_ERRORS, NULL);
+	stream = php_stream_open_wrapper((char *)fn, "rb", REPORT_ERRORS, NULL);
```

Comment:
The patch should be updated, too.

Thanks.

Reply:
Am not sure i understand what you mean :)

Reply:
libmagic is patched, the current patch is always contained in `ext/fileinfo/libmagic.patch`. The patch is made by diffing the patched vs. original sources every time something was changed, otherwise the change will not survive at the next upgrade.

thanks.

Reply:
I see. How should i do that? Reverting the libmagic patch in master fails.

```
$ patch -p0 -R --dry-run < libmagic.patch 
checking file libmagic/apprentice.c
checking file libmagic/ascmagic.c
checking file libmagic/cdf.c
checking file libmagic/cdf.h
checking file libmagic/cdf_time.c
checking file libmagic/compress.c
checking file libmagic/der.c
checking file libmagic/elfclass.h
checking file libmagic/file.h
checking file libmagic/fsmagic.c
checking file libmagic/funcs.c
Hunk #12 FAILED at 465.
1 out of 15 hunks FAILED
checking file libmagic/magic.c
checking file libmagic/print.c
checking file libmagic/readcdf.c
checking file libmagic/softmagic.c
Hunk #5 FAILED at 406.
Hunk #6 succeeded at 655 (offset 4 lines).
Hunk #7 succeeded at 679 (offset 4 lines).
Hunk #8 succeeded at 1193 (offset 4 lines).
Hunk #9 succeeded at 1361 (offset 4 lines).
Hunk #10 succeeded at 1476 (offset 4 lines).
Hunk #11 succeeded at 1560 (offset 4 lines).
Hunk #12 FAILED at 1687.
Hunk #13 FAILED at 1882.
3 out of 13 hunks FAILED
checking file libmagic/strcasestr.c
```

Reply:
The procedure is about getting libmagic of the same version as in ext/fileinfo/libmagic/magic.h and then runing diff . It's about updating the patch file, not about applying it.

Thanks.

Reply:
Updated the branch, thanks for the review.

Reply:
Pah, seems the patch should've been regenerated after PCRE2 migration, in first place. It's going to be different in lower branch. No worries then, i'll care about it. Should be done in 7.2 as well, as it could come to libmagic upgrade there. In this case still prefer your first variant, will generate libmagic.patch in both branches separately.

Thanks for checking, anyway :)

Reply:
So want me to push the change without the libmagic.patch right?

Reply:
Nope, no need to.